src/general/statistics_tracking/ReproductionStatistic.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Components;
+using DefaultEcs;
+using Newtonsoft.Json;
+
+public class ReproductionStatistic : IStatistic
+{
+    [JsonIgnore]
+    public StatsTrackerEvent LinkedEvent { get; set; } = StatsTrackerEvent.PlayerReproduced;
+
+    [JsonProperty]
+    public int TimesReproduced { get; set; }
+
+    [JsonProperty]
+    public Dictionary<Biome, int> ReproducedInBiomes { get; set; } = new();
+
+    [JsonProperty]
+    public Dictionary<string, int> ReproducedWithOrganelle { get; set; } = new();
+
+    public void RecordPlayerReporduction(in Entity player, Biome? biome)
+    {
+        TimesReproduced++;
+
+        var organelles = player.Get<OrganelleContainer>().Organelles?.Organelles.ToHashSet();
```

Comment:
This HashSet call here seems pretty useless. The organelles are looped just once and hashing the placed organelles to unique shouldn't even do anything.